tests/app/test_model.py

```diff
@@ -120,3 +120,26 @@ def test_has_statuses_should_accept_multiple_statuses(self):
             services = Service.query.has_statuses('published', 'disabled')
 
             assert_equal(services.count(), 2)
+
+    def test_purge_nulls(self):
+        service_with_nulls = {
+            'serviceName': 'Service with nulls',
+            'empty': None,
+            'serviceSummary': None,
+            'price': 'Not a lot'
+        }
+        same_service_without_nulls = {
+            'serviceName': 'Service with nulls',
+            'price': 'Not a lot'
+        }
+        with self.app.app_context():
```

Comment:
I don't think this test needs a persisted service in the database – we could do the same thing with ad-hoc `Service(data=service_with_nulls)` objects, which means there's no need to set up suppliers or any other database records